doc/connectivity/bluetooth/api/mesh/lcd_cli.rst

```diff
@@ -10,7 +10,8 @@ The Large Composition Data Client model was introduced in the Bluetooth Mesh Pro
 
 The Large Composition Data Client model communicates with a Large Composition Data Server model using the device key of the node containing the target Large Composition Data Server model instance.
 
-If present, the Large Composition Data Client model must only be instantiated on the primary element.
+If present, the Large Composition Data Client model must only be instantiated on the primary
+element.
```

Comment:
It seems to me that this change is not necessary?

Reply:
We shall follow 100 line width. But fine, I'll move it to a separate PR.

Reply:
In docs? It looks like we have many lines that are longer than just 100 in width? Anyway, I would not insist on a separate PR.

Reply:
See this: https://github.com/zephyrproject-rtos/zephyr/pull/62524****

Reply:
https://docs.zephyrproject.org/latest/contribute/guidelines.html#coding-style:

`The line length is 100 columns or fewer. In the documentation, longer lines for URL references are an allowed exception.`

Reply:
Ok. All good then :)